github/repos_pages_test.go

```diff
@@ -52,6 +52,34 @@ func TestRepositoriesService_EnablePages(t *testing.T) {
 	}
 }
 
+func TestRepositoriesService_UpdatePages(t *testing.T) {
```

Comment:
You could add a second test proving that when `CNAME` is empty, `null` is truly passed.
An example like that is `TestGitService_CreateTree_Delete` in `git_trees_test.go`.

Reply:
Added test.